teradatalabs/cdh5-hive/Dockerfile

```diff
@@ -59,6 +59,6 @@ EXPOSE 8032 8030 8031 8033 8088 8040 8042 8041 10020 19888
 EXPOSE 10000 9083
 
 # SOCKS PORT
-EXPOSE 1080
+EXPOSE 1180
```

Comment:
Nah, dammit. It's me who didn't add that port in the first place. Can you add it to hdp2.5-hive/Dockerfile? :)